README.md

```diff
@@ -39,12 +86,33 @@ Loaded templates: _templates
       inject: src/index.js
        added: src/components/Dropdown/README.md
        added: src/components/Dropdown/story.js
-      inject: src/stories.js
+      inject: src/stories/index.js
        added: src/components/Dropdown/test.js
 ```
 
 At any time you can see what generators we have configured using `hygen help`.
 
+### Customizing the theme with a custom build
```

Comment:
Seems like this section should be after the CSS-in-JS section since it builds off that info

Reply:
I meant for this to be a preface to both css in js and modules sections. I added a little more to clarify it.